components/pkg-export-kubernetes/src/bind.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::ArgMatches;
+use std::str::FromStr;
+
+use error::Error;
+
+#[derive(Debug, Clone)]
+pub struct Bind {
+    pub name: String,
+    pub service: String,
+    pub group: String,
+}
```

Comment:
Naming question for you... in the interest of having `Bind` mean only one thing in the Habitat codebase, is there another more descriptive name we could use here (like `K8sBind`, `KubeBind`, etc.)?

I know we're in the Kubernetes exporter and all, but before I got to this commit of the PR, for a second I was wondering why you'd need to be using `core::metadata::Bind`.

Reply:
I think in Rust (like most modern languages with the concept of module hierarchy), there is very little for ambiguity and adding such prefixes seems very much redundant to me. If you are in the code of the k8s exporter itself, then the context is very much obvious if you only read/write "Bind (even if sometimes it takes a few seconds before you recall what the current context is :-)). If you are using this API from another crate, you either have to use the module prefix with the type or if it's ambiguous, you must create an alias (use x.y.Bind as XyBind).

A k8s or kubernetes prefix would also be a bit misleading in this case since the bind is a concept of habitat, not k8s.

FWIW, on the lower (C) level (if you ever needed to interface with C for some reason), this would also imply double prefixes and much longer identifier names than they need to be.